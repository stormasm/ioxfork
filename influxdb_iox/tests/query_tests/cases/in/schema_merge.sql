-- IOX_SETUP: MultiChunkSchemaMerge

-- IOX_COMPARE: sorted
SELECT * from cpu;

-- Merge a subset
-- IOX_COMPARE: sorted
SELECT host, region, system from cpu;