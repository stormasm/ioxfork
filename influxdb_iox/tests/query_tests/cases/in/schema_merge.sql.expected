-- Test Setup: MultiChunkSchemaMerge
-- SQL: SELECT * from cpu;
-- Results After Sorting
+------+--------+--------+--------------------------------+------+
| host | region | system | time                           | user |
+------+--------+--------+--------------------------------+------+
| bar  | west   |        | 1970-01-01T00:00:00.000000250Z | 21.0 |
| foo  | east   |        | 1970-01-01T00:00:00.000000100Z | 23.2 |
|      | west   | 5.0    | 1970-01-01T00:00:00.000000100Z | 23.2 |
|      | west   | 6.0    | 1970-01-01T00:00:00.000000150Z | 21.0 |
+------+--------+--------+--------------------------------+------+
-- SQL: SELECT host, region, system from cpu;
-- Results After Sorting
+------+--------+--------+
| host | region | system |
+------+--------+--------+
| bar  | west   |        |
| foo  | east   |        |
|      | west   | 5.0    |
|      | west   | 6.0    |
+------+--------+--------+